docs/source/reference/_git-help.rst

```diff
@@ -1,91 +1,22 @@
-Using git for version control
+Using Git for version control
 =============================
 
-While version control is optional, it is _highly recommended [#]_ [#]_
```

Comment:
I'd like to keep the footnote references.  If we say highly recommended, _who says this and why?_  The footnotes are there to answer.  Move them to `instrument/_install_new_instrument.rst`.